README.md

```diff
@@ -1435,6 +1435,7 @@ _Related: [Monitoring](#monitoring)_
 - [Homer](https://github.com/bastienwirtz/homer) - A dead simple static homepage to expose your server services, with an easy yaml configuration and connectivity check. `Apache-2.0` `HTML5`
 - [Jmz HomeProxy](https://github.com/jmztaylor/homelab_proxy) - A simple and clean dashboard for self hosted services. `GPL-3.0` `PHP`
 - [LinkPage](https://links.zrd.sh) - LinkPage is a FOSS self-hosted alternative to link listing websites such as LinkTree and Campsite.bio. ([Source Code](https://github.com/rhnvrm/linkpage)) `BSD-2-Clause` `Go`
+- [LittleLink Custom](https://littlelink-custom.com/) - LittleLink Custom is an open source, highly customizable,  self-hosted alternative to Linktree, Manylink and other link sharing platforms with an intuitive, easy to use user/admin interface. ([Demo](https://demo.littlelink-custom.com/), [Source Code](https://github.com/JulianPrieber/littlelink-custom)) `GPL-3.0` `PHP`
```

Comment:
@JulianPrieber 
- double-space before `self-hosted`
- the description should describe what the software does. I suggest

> Open-source, customizable, self-hosted alternative to services like Linktree and Manylink with an intuitive, easy to use user/admin interface. LittleLink Custom allows you to link all your social media platforms easily accessible on one page.

Reply:
Thanks @nodiscc, resolved with https://github.com/awesome-selfhosted/awesome-selfhosted/pull/2974/commits/454c22ef8b35f28ebefaa0a98e24db079f947e8e